mitodl/micromasters
# Incorporated update homepage design

#### What are the relevant tickets?

this closes #97 

#### What's this PR do?

Implements the latest design for the home page. You can see the unincorporated template over [here](https://github.com/mitodl/micromasters-templates)

#### Where should the reviewer start?

Oh god, everywhere.

major changes:

- changed webpack config (sharing a file between dev and prod)
- added new webpack entry point (to bundle stylesheets for the homepage)
- updated html for homepage (`/cms/templates/cms/home_page.html`)
- pulled in vendor css and vendor JS (under `/static/scss/public` and `/static/js/public`, respectively)
- added new image assets
- added new navigation theme to the dashboard side of things as well

I tried to cut down on what we needed to check in to version control as much as possible. Unfortunately a number of things are used here that I couldn't find npm packages for (like the `raty` plugin that shows the rating stars, and a bunch of css).

#### How should this be manually tested?

Pull it down, make sure everything looks good. If you can think of any ways to cut the diff down further that would be :100: 

#### Any background context you want to provide?

Second version of the homepage. Tried to keep the diff as small as possible.

#### Screenshots (if appropriate)

![micromasters_home_page](https://cloud.githubusercontent.com/assets/6207644/14791900/dfbebaba-0ae5-11e6-93d9-df9dcb5bf2b8.png)
